widgets/show-dataviz-widget.php

```diff
@@ -67,6 +74,8 @@ public function update( $new_instance, $old_instance ) {
 
   $instance = array();
   $instance['id'] = ( ! empty( $new_instance['id'] ) ) ? strip_tags( $new_instance['id'] ) : '';
+  $instance['hide_description'] = ( ! empty( $new_instance['hide_description'] ) ) ? strip_tags( $new_instance['hide_description'] ) : "false";
```

Comment:
yep .. i first done it with boolean but when the data return `false` the shortcode take empty value instead of `false`. That's why i changed them to strings. May be should i change them with 0, 1 ?